### ISSUE TITLE
Can't preview MarkDown file in vscode.

### ISSUE THREAD

**Issue by xsyww** [Microsoft/vscode#30584]
- VSCode Version: Code 1.14.0 (b8cd5b9556d8b70ea560d35b903422363f6c5c40, 2017-07-10T18:36:31.751Z)
- OS Version: Windows_NT ia32 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
code-gnu-global|aus|0.2.2
python|don|0.6.7
vscode-great-icons|emm|2.1.9
vscode-JS-CSS-HTML-formatter|lon|0.2.3
cpptools|ms-|0.12.0
csharp|ms-|1.11.0
debugger-for-chrome|msj|3.1.6

---

Steps to Reproduce:

1. open vscode and create a .md file
2. write content "## My Title " and open preview window, and it's show nothing!

Reproduces without extensions: Yes/No

**Comment by chrmarti** [comment]
Cannot reproduce. Could you check if there is anything in the console? (Help > Toggle Developer Tools)

**Comment by stevechae** [comment]
I was experiencing this since 1.14.0 as well although sporadically. I opened an existing `.md` and the Preview showed nothing. It was fairly easy to reproduce soon after the update but it seems difficult to do so now oddly enough. I did look at Help > Toggle Developer Tools while I was having the issue but didn't see any errors or warnings regarding Markdown. 

One thing I did though, is to uninstall [Markdown + Math plugin](https://www.google.ca/search?q=visual+studio+code+markdown+math&oq=visual+studio+code+markdown+math&aqs=chrome..69i57j69i60l4j69i61.3038j0j4&sourceid=chrome&ie=UTF-8) but I'm not sure if this had anything to do with it (like I said, there was nothing notable in the Developer Tools console).
